application/helpers/expressions/em_core_helper.php

```diff
@@ -1849,7 +1877,7 @@ private function RDP_RunFunction($funcNameToken, $params)
                     switch ($argsPassed) {
                         case 0:
                             if (!$this->RDP_onlyparse) {
-                                $result = $funcName();
+                                $result = call_user_func($funcName);
```

Comment:
Any difference, really?

Reply:
Didn't test, but i dislike to use $funcName like this … and unsure it don't make difference with complete class …